apps/docs/pages/guides/functions/examples/upstash-redis.mdx

```diff
@@ -0,0 +1,91 @@
+import Layout from '~/layouts/DefaultGuideLayout'
+
+export const meta = {
+  id: 'examples-upstash-redis',
```

Comment:
We don't use this anymore (still need to remove it from all the other mdx files).
```suggestion
```